coalesce/opportunities/models.py

```diff
@@ -0,0 +1,25 @@
+import uuid
+
+from django.db import models
+from django.contrib.postgres.fields import ArrayField
+
+class Opportunity(models.Model):
+
+    background_check_requirements = models.TextField()
+    commitment_type = models.TextField()
+    datetime = models.DateTimeField(help_text="Event date", blank=True)
+    description = models.TextField()
+    clothing_requirements = models.TextField()
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
I think id should be automatically generated with anything that inherits `models.Model`